packages/sdk/src/types/runtime/actor.ts

```diff
@@ -282,9 +293,12 @@ export class Actor implements ActorLike, Patchable<ActorLike> {
 	}
 
 	/**
-	 * Loads assets from a glTF file, populates the container with the assets, and spawns
-	 * @param container
-	 * @param options
+	 * @deprecated
+	 * Use [[AssetContainer.loadGltf]] and [[Actor.CreatePrefab]] instead.
+	 * @param container The asset container to load the glTF assets into
+	 * @param options.uri A URI to a .gltf or .glb file
+	 * @param options.colliderType The type of collider to add to each mesh
+	 * @param options.actor The initial state of the actor
 	 */
 	public static CreateFromGltf(container: AssetContainer, options: {

```

Comment:
Great idea to deprecate methods rather than delete them, but don't deprecate `CreateFromGltf`. It's a convenience method we should keep.